### ISSUE TITLE
Unable to run test suites

### ISSUE THREAD

**Issue by dangerdogz** [spree-contrib/spree_active_shipping#238]
I am unable to execute the test suite from a clean pull of spree_active_shipping

Running "bundle exec rake test_app" fails everytime because assets are not found

Sprockets::FileNotFound: couldn't find file 'spree/backend/spree_active_shipping'
  (in /Users/lawebshopimac/RubymineProjects/spree_active_shipping/spec/dummy/vendor/assets/javascripts/spree/backend/all.js:11)

Can't find much about similar problems so I expect that I am doing something wrong but I can't figure it out

How do I test the gem in isolation?

Thanks

**Comment by jspizziri** [comment]
You need to run `bundle install` to install the required gems before you rum `rake test_app`.

Please reference this https://github.com/spree-contrib/spree_active_shipping#testing

**Comment by dangerdogz** [comment]
Did not specify, my mistake, but yes, I ran through the steps and I have a brand new gemset with no cache or conflicting version/libs

There is nothing else to do? Clone the repo, run bundle, bundle install, rake test_app? 

I'm on the 3-0-stable branch if it changes anything. I did see some bugs though, mainly related to ActiveShipping namespacing where the master doesn't have the proper fixes but 3-0-stable does.
